### DISCUSSION TITLE
dandihub - question about persistence and source of truth for files

### DISCUSSION THREAD

**Discussion by magland** [dandi/helpdesk#104] [General]
Hi, I'm not sure I understand dandihub. I started a server and I can view and run a bunch of notebooks. What happens if I modify these files? (Of course the notebooks will be modified if I run them). Will the changes persist? Will other people see the changes? I shut down my server and started a new one, and the files were still modified. I'm confused about what the source of truth for all these files.

Thanks!
Jeremy

**Comment by satra** [comment]
others will not see any changes you made. your home directory is personal. however if you place anything in `/shared` anyone can read/write/delete it.  the source of truth is the [example-notebooks](https://github.com/dandi/example-notebooks) repo in the dandi organization. if you delete the notebooks folder on the hub it will be recreated when you log out and log back in. also since it's a git repo one could resync it or stash the local changes.

these persistence settings will be adjusted this coming year and be made clearer.

**Comment by magland** [comment]
Great, thank you.
